### ISSUE TITLE
Can we revive `test_avmc`?

### ISSUE THREAD

**Issue by junkmd** [enthought/comtypes#572]
Modern packaging methods were introduced in #556 and #557. Until then, the code base in `setup.py` was old, such as not having `distutils` replaced by `setuptools` until Python 3.12 support.

As a result, the cognitive load when reading `setup.py` has been reduced, making `class test(Command):` easier to read and clearly showing what it was doing. (I confess I hadn't properly read it before since it wasn't hooked when running tests with the `unittest` command.)

After skimming through it, I understand that it enables the following options:
- `tests=` or `t`: Specify the names of tests to run, separated by commas
- `use-resources=` or `u`: Add `resource` to be used during testing
- `refcounts` or `r`: Conduct tests that depend on `sys.gettotalrefcount` in a debug-built Python

Tests executed with these options register either `AvmcIfc.dll` or `AvmcIfc_x64.dll`.

In other words, if we create a GHA workflow to mimic the tests executed from this setup, we might be able to revive `test_avmc`, which remained broken in #298 (#267).

`test_avmc` is not a high priority for me at the moment to actively push for a discussion, so this is more of a memo.
However, if any community members are interested, I would be happy to cooperate.
